lib/themes/dosomething/paraneue_dosomething/js/utilities/ApiClient.js

```diff
@@ -18,7 +18,8 @@ class ApiClient {
     this.host = host;
     this.port = port === '' ? null : port;
     this.version = `/api/${version}/`;
-    this.url = `//${this.host}${this.port ? ':' + this.port : ''}${this.version}`;
+    this.hostname = `//${this.host}${this.port ? ':' + this.port : ''}`;
```

Comment:
would it be better to call this `this.domain`? :shrug: I feel like `host` and `hostname` seem like the same thing.